### PR TITLE
GTEST/UCP/SOCKADDR: fix test_ucp_sockaddr.listen_inaddr_any

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
fix address storage

## Why ?
bad commit is c1b9af1d923367177a692b92588a7ec53aba09d2

### CODE REVIEW

File: test/gtest/ucp/test_ucp_sockaddr.cc [hunk 1]
```diff
@@ -598,11 +598,17 @@ UCS_TEST_P(test_ucp_sockaddr, concurrent_disconnect_bidi) {
...
+    ucs::sock_addr_storage test_addr(m_test_addr);
     m_test_addr.reset_to_any();
 
     UCS_TEST_MESSAGE << "Testing " << m_test_addr.to_str();
 
     start_listener(cb_type());
+    /* get actual listening port */
+    test_addr.set_port(m_test_addr.get_port());
+    /* restore address */
+    m_test_addr = test_addr;
```
Comment (alinask) [inline test/gtest/ucp/test_ucp_sockaddr.cc]: I would restore at the end of the test but that's ok too.

Comment (evgeny-leksikov) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: it should be restored before connect

Comment (alinask) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: m_test_addr is used both for listening on and for connecting to?

Comment (evgeny-leksikov) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: yes

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 1 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 04-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/cc0aa5118e4737e90347d1d1331b22ca8fe8e08a/_main) | :question: ABORTED
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/353b7e18f3f2e7ded38afb7d63322250a00b9ff3/hpc-test-node-legacy_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/5978995592a275ff5cf84492bb52991112ca0944/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/cb6a01202b9c9be2e4a1984072cca7e33aebec8f/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/7d2820a672128525ab9ebd913b701574ac0ea3d4/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/7608a223d617670c6d24c310888b960a860e2ee5/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/ecc5de7c69bcfff8172517b3f923f924e2e7e5b5/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/d0a1966eed706b6193c4e3e030b6a156656324fd/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/a7da48554963c1cdbb58a9f0a34a1cc2bb27fb05/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/ff999410944e75d089c0d15f419afc5386f6e6e8/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/f4e87aeff56a656aff158112a204f20da977f3b9/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/a48052fc1a90e21be2a895bde57cd5fd1efe222f/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/f28be8210a98c78bb833a905d9331482264916fa/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/367127982e21cba127153422a1b6df9c57b5e2db/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/4efdd6dbc0f21ba84f1c05ddaa6f3b724fbb2917/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/64ec483f7781ea29ef31928e4711df94759cf50f/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/ff680554bc5a68f2582f6d010133e29b4dda4f17/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/66613ed3316a3fcbbaa4cbff6602832ea0a432dc/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/579e9a5526967b88081467a889d6a87d9554d55f/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/a976a3de98c8be55716bcc9d0005adf82baba1fd/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/8c48e41e8575af3713d98b1cadb9172534067c4e/hpc-test-node-new_W3) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/6d214d837caee033e494a70a8ba0924b3e5fc716/r-vmb-ppc-jenkins_W0) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/777e72984bfbe3728ae95c7fcf63a0ddb2e4bb06/r-vmb-ppc-jenkins_W1) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/b169c4e36f497c9d2fd3ec1a16d1f3f128a8d659/r-vmb-ppc-jenkins_W2) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/cfcf058c5add602b1360dfcf33644b9b/raw/fd01c73716b541953e28e8ebc26ccb5121092356/r-vmb-ppc-jenkins_W3) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 04-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/9c38193fe2be607b01da5fe40bfc74d97242a022/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/b46f785acd43efe79abd91f7a6d148d2d5f171ff/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/d0daecc7245a867b3d4c7f2177d211b29d4c4ae7/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/dc6e78a14d4c0a005fc710963b37df04206d4a64/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/622827244c3ddca686faa6710652b91207afe1f6/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/4510ac8f0b70f86d39ae3912fed088c8a508c07e/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/7fb9ae68113f351ef9dbe7db76239bfbb2872b7d/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/fab43060a9d8b640e9cf6556b10789c40211f8be/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/aa765a5662c1e23f2381603662950df4132c1835/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/24df4d61b864e79279061f90652501589f60e337/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/aa19c3f58e2089f17f856fd4cf3837a31191fe57/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/3bdb35264bee329b13a93b2d3e7a20791bfae0fe/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/a035daaad98c8d9ad3d2268a9fd3ab97e7fec0bf/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/d56f388839cf0194d2e65128a69f8c2b6d41a9a1/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/d262382d115b2d34a8ae4b7a97ed224a81833bc0/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/bfbb62c8dff1336d0bf3553b953bad9f92b0dbca/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/e4bc1a4f4d123ac9e9cab2e1beac8d8fc87cb5ac/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/569bc14e53421544bc682011f20a649dce1b7b29/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/4502fb9d8f352115d5898d48ca58f67c27b2ef26/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/5273a9c31922b4befd61efe1529630a90b0b89f3/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/216d4b06a191904ec506c0c794a61781f07ecfd0/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/71bc28629dab826086b7d71c243d31e25068a886/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/8f3c604db5ddb2f58838981c3b50b2a7dca778de/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/3111c2147173b800279a9d9bd45320a4c1bc6903/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5c080eb74eb06bed172f08071e2be0a3/raw/d151ca11d92ad98d565c17d6872643349a95b1dd/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: test/gtest/ucp/test_ucp_sockaddr.cc [hunk 2]
```diff
@@ -598,11 +598,17 @@ UCS_TEST_P(test_ucp_sockaddr, concurrent_disconnect_bidi) {
...
 UCS_TEST_P(test_ucp_sockaddr, listen_inaddr_any) {
+    /* save testing address */
+    ucs::sock_addr_storage test_addr(m_test_addr);
     m_test_addr.reset_to_any();
 
     UCS_TEST_MESSAGE << "Testing " << m_test_addr.to_str();
 
     start_listener(cb_type());
+    /* get actual listening port */
+    test_addr.set_port(m_test_addr.get_port());
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_sockaddr.cc]: why is this line needed? because the port could change if busy? pls expand comment..

Comment (evgeny-leksikov) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: all the UCP tests start with port 0, so UCP will get random port, when listener is started test queries it to get actual port number. the listener creation can fail with BUSY only if different UCT CMs can't share the same port. Lets say, rdma-listener selected the port where tcp-listener can't be started.

I think these lines already explain what you requested:
https://github.com/openucx/ucx/blob/cefa5e20cb9810d8fccf3d117957b336c3b06178/test/gtest/ucp/test_ucp_sockaddr.cc#L154
https://github.com/openucx/ucx/blob/cefa5e20cb9810d8fccf3d117957b336c3b06178/test/gtest/ucp/test_ucp_sockaddr.cc#L187

Comment (yosefe) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: i mean, pls replace `/* get actual listening port */`
with smth like `/* initial port could be busy, so get the actual port which was selected */`

Comment (evgeny-leksikov) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: but it's incorrect, initial port of `m_test_addr` is 0. this value is stored in begin of the test.
would "get the actual port which was selected by listener" be clear?

Comment (yosefe) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: yes, ok

Comment (evgeny-leksikov) [reply test/gtest/ucp/test_ucp_sockaddr.cc]: force-pushed since change is small

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 05-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/b3bcc99188d67457477471ab1bc64104895affa7/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/3a2ac7ebd33f1f1d99226ea484e2b05806f0063f/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/c88da6b299baed8cc404b0026be813d3bce47ed2/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/899335876c64218e13879be790113017228818ce/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/d424c7f97e19a415c3e68df02f065beabee165e7/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/fc6bbba06a18a7ac37aceb3bddfc7487aa0b7b57/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/12976a78be5bcf6a165ea5527f6a5b80b35d0b44/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/eea284c124520404f2ba8e3dd310ac558cb846d3/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/e4fc226bff347dbdf95258d3788c0a0aa8419f4c/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/f64d77c0348372ba4ca7cf5a9e68f217fc06193d/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/2a6a74cefc474a973ebdddb06d18c22cc021e94c/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/905469ba2c16fcd07737f0879ee67017372a82a5/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/da3ade63accba3074a78214d64055bfa416b60c6/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/337d1b0d47b6eaae0adfbd5c85c4d36e9cb92630/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/c4e400aaaab2f7c945ff12026e80b36ad9b59eb7/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/14976bf44f4318c9163e986ddc63c279ba036907/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/01333fe78fe40f73adc659c7ef4b3f08a2d3b997/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/6056763f57ea88ba2beb9208f9bcbe26be298206/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/d05002c500546a3ee4519ce2249c55aa1e029707/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/7fe026a280d1701d26fb590c487482ef891b3b82/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/989b3a8b18a126c05199d9b4fb007cd7e8593fd6/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/9b819401374751b7d87707cf49d2156aff50d1bb/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/a9d1cf277b8ecf8f4a806e854b810f9c48586011/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/302a2ac9f71ec48d652ddd8b05d491ab95648d96/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/547c963c34e42039c5d07d13a874c64a/raw/d3c8041f5bcbd55398f2b176b65ea1efd629fab8/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
